### ISSUE TITLE
[Bug] Extension have no effect in Linux

### ISSUE THREAD

**Issue by Armaldio** [d-kimuson/ts-type-expand#23]
Whatever type I select, nothing never appeared inside the panel

Here is a single example:

https://user-images.githubusercontent.com/3089715/131960583-889d77c4-a783-485e-adab-91c3f33f329f.mp4

VSCode Typescript version: 4.4.2
Local Typescript version: 4.1.6

**Comment by d-kimuson** [comment]
Hi, thanks for the report.

First, please make sure that one of the following is satisfied to enable this extension

- tsconfig.json is placed in the workspace root
- `ts-type-expand.tsconfigPath` is set to the correct relative or absolute path to tsconfig.json

From today's v0.0.9 release, if tsconfig.json fails to load, an error message will be displayed so that you can determine if the loading was successful.

If that does not solve the problem, I will need to see the source code and directory structure to create a minimal repository that can reproduce the problem. Please also give me some information about the device you are using.

Thank you very much.

**Comment by Armaldio** [comment]
Hello,

> tsconfig.json is placed in the workspace root
You can see that in the screenshot
I can also confirm I'm on v0.0.9 and have no error message displayed

![image](https://user-images.githubusercontent.com/3089715/132172518-9df8c206-bbd6-45b3-9a96-129ee7502102.png)

Also, maybe related, when I open the command palette and select the restart command, I get this error message: 
(fr)
```
La commande 'Restart (ts-type-expand)' a entraîné une erreur (command 'ts-type-expand.restart' not found)
```
(en, translated)
```
The command 'Restart (ts-type-expand)' resulted in an error (command 'ts-type-expand.restart' not found)
```

I'll try to setup a minimal repro as my files are private, but my current project is already pretty small.

Thanks for your time

**Comment by Armaldio** [comment]
![image](https://user-images.githubusercontent.com/3089715/132173020-9d47f8e2-c21d-43b8-93fd-f1ae2cfabd29.png)

**Comment by d-kimuson** [comment]
@Armaldio 
Sorry for the delay.
I found that the extension doesn't seem to work in Linux environment.
I haven't figured out the cause yet, so it may take some time to resolve.
I will let you know when I release a resolved version.

**Comment by Armaldio** [comment]
No problem :+1: 

Please let me know if I can help in any way

**Comment by d-kimuson** [comment]
@Armaldio 
I found that dependency resolution was failing on case-sensitive filesystems.
We've fixed it in v0.0.11 and confirmed that it can be activated on Ubuntu machines that had the same problem.
Please try the latest version!

**Comment by Armaldio** [comment]
It worked perfectly !
Thanks a lot
